add descriptions to severless cr spec (#238)
kyma-project/serverless-manager: docs/user/resources/06-20-serverless-cr.md

| Parameter                                 | Type    | Description |
|-------------------------------------------|---------|-------------|
| **dockerRegistry**                        | object  |             |

> | **dockerRegistry.&#x200b;enableInternal** | boolean | When set to `true` the internal Docker registry is enabled           |

Reviewer: ```suggestion
| **dockerRegistry.&#x200b;enableInternal** | boolean | When set to `true`, the internal Docker registry is enabled           |
```

| **dockerRegistry.&#x200b;secretName**     | string  | Secret used for configuration of the Docker registry            |
| **eventing**                              | object  |             |
| **eventing.&#x200b;endpoint** (required)  | string  | Used Eventing endpoint            |
| **tracing**                               | object  |             |
| **tracing.&#x200b;endpoint** (required)   | string  | Used Tracing endpoint            |

> | **targetCPUUtilizationPercentage**        | string  | Sets a custom target threshold for CPU utilization            |

Reviewer 2: ```suggestion
| **targetCPUUtilizationPercentage**        | string  | Sets a custom CPU utilisation threshold for scaling function pods            |
```

> | **functionRequeueDuration**               | string  | Sets the requeue duration for Function. By default, the Function associated with the default configuration will be requeued every 5 minutes            |

Reviewer: ```suggestion
| **functionRequeueDuration**               | string  | Sets the requeue duration for Function. By default, the Function associated with the default configuration is requeued every 5 minutes            |
```
